src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -126,6 +133,14 @@ export const tooltipBottomLeftWrap = (): React.ReactElement => (
     </Tooltip>
   </div>
 )
+tooltipBottomLeftWrap.parameters = {
+  happo: {
+    waitFor: () =>
+      document.querySelector(
+        '.usa-tooltip__body.usa-tooltip__body--right.usa-tooltip__body--wrap.is-visible.is-set'
```

Comment:
Depending on the screen size that happo is taking a screenshot for, the tooltip position may use a different default position, which would mean different different classes than what you expect to find. That probably limits us to doing something like:

```suggestion
        '.usa-tooltip__body.usa-tooltip__body--wrap.is-visible.is-set'
```

Or more generically using waitFor at the top level with `'.usa-tooltip__body.is-visible.is-set'`.

A lot of the flaky diffs _are_ on the ios-safari screenshots, which usually forces the alternate position of the tooltip.

Reply:
Fixed!